### PR TITLE
Create MAINTAINERS.md

### PR DESCRIPTION
**Pull request by BVrolyk**
Define the Helm project as having one type of maintainer. Identify the maintainers and the BDFL.

### CODE REVIEW

**Comment by gabrtv** [comment]
Pinging @carmstrong @mboersma @technosophos for review on this one.

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* Help to shape the Helm roadmap and lead efforts to accomplish roadmap milestones
+* Participate actively in feature development and bug fixing
+* Answer questions and help users in IRC
+
+The current maintainers of Helm:
+* Matt Butcher - <mbutcher@engineyard.com> ([@technosophos](https://github.com/technosophos))
+* Matt Boersma - <mboersma@engineyard.com> ([@mboersma](https://github.com/mboersma))
+* Chris Armstrong - <carmstrong@engineyard.com> ([@carmstrong](https://github.com/carmstrong))
+
+No pull requests can be merged until at least two maintainer signs off with an
```
Comment (mboersma) [inline MAINTAINERS.md]: This should say "two maintainer**s** sign off".

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+The duties of a maintainer include:
+* Classify and respond to GitHub issues and review pull requests
+* Help to shape the Helm roadmap and lead efforts to accomplish roadmap milestones
+* Participate actively in feature development and bug fixing
+* Answer questions and help users in IRC
+
+The current maintainers of Helm:
+* Matt Butcher - <mbutcher@engineyard.com> ([@technosophos](https://github.com/technosophos))
+* Matt Boersma - <mboersma@engineyard.com> ([@mboersma](https://github.com/mboersma))
```
Comment (mboersma) [inline MAINTAINERS.md]: Can this be `<mboersma@deis.com>` instead?

**Comment by mboersma** [comment]
So perhaps we have not formalized this in the deis docs, but all Engine Yard employees working on Deis are considered contributing maintainers for reviewing and LGTM purposes, and I think the same has been true for Helm, right?

**Comment by technosophos** [comment]
I've been just linking to the `deis/deis` maintainers file unless there is some reason to have a special one. Otherwise we'll just end up maintaining pretty much the same file in every repo.

**Comment by bacongobbler** [comment]
> I've been just linking to the deis/deis maintainers file unless there is some reason to have a special one.

As an example: https://github.com/technosophos/prototype-repo/blob/master/MAINTAINERS.md

**Comment by carmstrong** [comment]
> but all Engine Yard employees working on Deis are considered contributing maintainers

We may just want to start linking to the GitHub teams themselves, assuming the public can view members.

**Comment by gabrtv** [comment]
Closing in favor of #172.

**Pull request closed without merging**